src/openfermion/functionals/get_one_norm.py

```diff
@@ -0,0 +1,135 @@
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+"""
+Function to calculate the 1-Norm of a molecular Hamiltonian
+(in the restricted formalism) after fermion-to-qubit transformation
+from a MolecularData class. See https://arxiv.org/abs/2103.14753
+for more information on the 1-norm.
+"""
+
+import numpy as np
+
+
+def get_one_norm_mol(molecule, no_constant=True):
```

Comment:
do not overload the inputs like this.  I understand you are dispatching off the boolean input but just have two different functions.  This burdens the user with trying to remember what are valid inputs.

Reply:
I'm not sure that this is being overloaded anymore? @Emieeel has split this into different functions, and the 'molecule' input should be identical for the two.

However, I think the docs need to be updated, as currently you are referring to 'constant', 'one_body_integrals' and 'two_body_integrals' which are no longer taken as input here.

Might also want to reference the two different functions in the docstring - saying something like 'Returns the 1-norm of a Hamiltonian contained in a MolecularData class (to calculate directly from one and two-body integrals refer to the 'get_one_norm_int' function).

Also, if I understand correctly, the '_get_one_norm' and '_get_one_norm_woconst' function literally differ by whether or not the constant is added, right? If so, you could get rid of the _get_one_norm function, and have a workflow like:

```
one_norm = _get_one_norm_woconst(molecule.one_body_integrals, molecule.two_body_integrals)
if add_constant:
    one_norm += abs(molecule.nuclear_repulsion)
return one_norm
```